public/background.js

```diff
@@ -13,21 +13,24 @@ chrome.runtime.onInstalled.addListener(function(details) {
         ]);
     });
     if (details.reason == 'install') {
-        chrome.tabs.create({ url: chrome.extension.getURL('welcome.html') }, function() {});
-        localStorage.setItem('notifications', true);
+        chrome.tabs.create({ url: chrome.runtime.getURL('welcome.html') }, function() {});
+        chrome.storage.local.set({'notifications': true});
```

Comment:
manifest v3 is built on a service worker, rather than the background page used in mv2. Therefore localStorage doesn't exist. moved to the chrome.storage

Reply:
How does it work when people migrate? will they see their items there?